ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -32,6 +32,9 @@ elif [ -f "${SHARED_DIR}/user_tags_sa.json" ]; then
 elif [ -f "${SHARED_DIR}/xpn_min_perm_passthrough.json" ]; then
   echo "$(date -u --rfc-3339=seconds) - Using the IAM service account of minimal permissions for deploying OCP cluster into GCP shared VPC..."
   export GOOGLE_CLOUD_KEYFILE_JSON="${SHARED_DIR}/xpn_min_perm_passthrough.json"
+elif [ -f "${SHARED_DIR}/ipi-xpn-no-fw-no-dns-sa.json" ]; then
```

Comment:
Is this sa also a min_perm_passthrough sa?

Reply:
Yes, this SA (SA-2) has even less permissions than the SA (SA-1) in xpn_min_perm_passthrough.json. 

The permissions **both SA-1 and SA-2 have**, in the service project (openshift-qe):
- roles/compute.admin
- roles/compute.loadBalancerAdmin
- roles/dns.admin
- roles/iam.roleViewer
- roles/iam.securityAdmin
- roles/iam.serviceAccountAdmin
- roles/iam.serviceAccountKeyAdmin
- roles/iam.serviceAccountUser
- roles/resourcemanager.tagUser
- roles/storage.admin

The permissions **both SA-1 and SA-2 have**, in the host project (openshift-qe-shared vpc):
- roles/compute.networkUser

The permissions **only SA-1 has**, in the service project (openshift-qe):
- roles/file.editor
- roles/resourcemanager.tagAdmin
_Note: The first is used by filestore instance provision, and the second is used to check userTags during post-installation check (e.g. workflow `cucushift-installer-rehearse-gcp-ipi-user-labels-tags-filestore-csi`)._ 

The permissions **only SA-1 has**, in the host project (openshift-qe-shared vpc):
- projects/openshift-qe-shared-vpc/roles/dns.networks.bindPrivateDNSZone
_Note: The permission is to bind private zone to the shared VPC._

Reply:
thanks for the details, can we update the sa name to add `min_perm_passthrough` keyword?

Reply:
Sorry, you mean the service account name, or the its json file name?

Reply:
this json file name

Reply:
Updated the json file into `xpn_byo-hosted-zone_min_perm_passthrough.json`, thanks!

Reply:
Along with renaming the gcp provision step into `gcp-provision-xpn-byo-hosted-zone-min-perm-passthrough-serviceaccount`.